src/calendar/calendar.style.tsx

```diff
@@ -0,0 +1,132 @@
+import { Color } from "src/color";
+import { ChevronLeftIcon } from "@lifesg/react-icons/chevron-left";
+import styled, { css } from "styled-components";
+import { TextStyleHelper } from "../text/helper";
+import { View } from "./types";
+
+interface HeaderDropdownProps {
+    view: View;
+}
+
+interface CalendarViewProps {
+    children: JSX.Element[];
+    show: View;
+}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const Container = styled.div`
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 41rem;
+    background-color: ${Color.Neutral[8]};
+    border: 1px solid ${Color.Neutral[5]};
+    height: 25.188rem;
+    border-radius: 0.75rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+`;
+
+export const HeaderDropdown = styled.div<HeaderDropdownProps>`
+    display: flex;
+    justify-content: center;
+    margin-bottom: 0.938rem;
+
+    svg {
+        margin-left: 0.625rem;
+        color: ${Color.Neutral[3]};
+        transition: transform 250ms ease-in-out;
+    }
+
+    ${(props) => {
+        switch (props.view) {
+            case "Month":
+                return css`
+                    & > :first-child {
+                        display: none;
+                    }
+
+                    & > :last-child svg {
+                        transform: rotate(180deg);
+                    }
+                `;
+            case "Year":
```

Comment:
identical to the previous case, so can merge

Reply:
previous case??

Reply:
ohh.. the prefix $?

Reply:
I meant in the switch case, Month and Year are handled the same way

so actually you can do
```
switch (...) {
  case "Month":
  case "Year":
    return css``
```

Reply:
Opps. Yes. they are same styling lol.